app/notification_push.go

```diff
@@ -102,6 +120,30 @@ func (a *App) sendPushNotification(post *model.Post, user *model.User, channel *
 	return nil
 }
 
+func (a *App) getChannelFromUserId(userId string) chan PushNotification {
```

Comment:
This could be made a method of the `PushNotificationsHub`. That would also make the name much clearer since I initially expected it to return a `model.Channel`.

Would it make sense to move any of these other methods onto the `PushNotificationsHub`?

Reply:
Also, this is the only one of these changes that I feel strongly about just because the name of the method is misleading

Reply:
Yes, this is very confusing because mattermost channel is one thing and go channels are other thing. Maybe something like `getGoChannelFromUserId` and probably put it in the `PushNotificationsHub` makes a lot of sense too.